example/factories.ts

```diff
@@ -52,7 +52,7 @@ export const fileStreamingEndpointsFactory = new EndpointsFactory({
   config,
   resultHandler: createResultHandler({
     getPositiveResponse: () => ({
-      schema: ez.file().buffer(),
+      schema: ez.file("buffer"),
```

Comment:
⚠️ breaking

Reply:
resolved in [03c7f20](https://github.com/RobinTail/express-zod-api/pull/1442/commits/03c7f2060a353d28c8676ec635a68ff4bb43c767)